Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3
+
+WORKDIR .
```

Comment:
Changing this to something like `/app` would help separate the app files from the root.